specs/sharding/beacon-chain.md

```diff
@@ -539,7 +539,7 @@ def update_pending_votes(state: BeaconState, attestation: Attestation) -> None:
     participants = get_attesting_indices(state, attestation.data, pending_header.votes)
     participants_balance = get_total_balance(state, participants)
     full_committee = get_beacon_committee(state, attestation.data.slot, attestation.data.index)
-    full_committee_balance = get_total_balance(state, full_committee)
+    full_committee_balance = get_total_balance(state, set(full_committee))
```

Comment:
Maybe we should introduce some type for an ordered-set in the spec. The committee members each have an index, and are unique, at the same time. The total-balance computation is order-independent though, so a `set` is fine here.